INSTALL_linux_xeon.md

```diff
@@ -0,0 +1,83 @@
+# Setup Guide for Linux with Xeon-Series CPU
+
+1. [Installation](#installation)
+2. [Start the Service](#start-the-service)
+
+## Installation
+
+### Download Langchain-Chatchat
+
+Download the Langchain-Chatchat with IPEX-LLM integrations from [this link](https://github.com/intel-analytics/Langchain-Chatchat/archive/refs/heads/ipex-llm.zip). Unzip the content into a directory, e.g. `/home/arda/Langchain-Chatchat-ipex-llm`. 
+
+### Install Prerequisites
+
+We recommend to use Conda to manage the running environment. For how to install Conda on your Linux distributions, please visit [here](https://conda.io/projects/conda/en/latest/user-guide/install/linux.html).
+
+### Install Python Dependencies
+
+#### 1. Create a Conda Environment
+Run the following commands to create a new python environment:
+```bash
+conda create -n ipex-llm-langchain-chatchat python=3.11
+conda activate ipex-llm-langchain-chatchat
+```
+
+#### 2.  Install `ipex-llm` 
+```bash
+pip install --pre --upgrade ipex-llm[all]
+pip3 install torch==2.1.2 torchvision==0.16.2 torchaudio==2.1.2
+```
+
+#### 3. Install Langchain-Chatchat Dependencies 
+Switch to the root directory of Langchain-Chatchat you've downloaded (refer to the [download section](#download-langchain-chatchat)), and install the dependencies with the commands below. **Note: In the example commands we assume the root directory is `/home/arda/Langchain-Chatchat-ipex-llm`. Remember to change it to your own path**.
+```bash
+cd /home/arda/Langchain-Chatchat-ipex-llm
+pip install -r requirements_ipex_llm.txt 
+pip install -r requirements_api_ipex_llm.txt
+pip install -r requirements_webui.txt
+# Due to an known issue, run with Llama-2 model
+# will require to install transformers==4.34.0 to generate the correct result.
+pip install transformers==4.34.0
```

Comment:
We may remove it when this bug is fixed. For now, we may mention in comment that this known issue is on CPU?